mdit_py_plugins/admon/index.py

```diff
@@ -128,7 +132,7 @@ def admonition(state: StateBlock, startLine: int, endLine: int, silent: bool) ->
     state.md.block.tokenize(state, startLine + 1, next_line)
 
     token = state.push("admonition_close", "div", -1)
-    token.markup = state.src[start:pos]
+    token.markup = markup
```

Comment:
Can you add a test, that checks that the markers are correctly captured in the token stream, something like:

```python
def test_plugin_parse(data_regression):
    md = MarkdownIt().use(admon_plugin)
    tokens = md.parse(
        dedent(
            """\
    ??? note
        content
    """
        )
    )
    data_regression.check([t.as_dict() for t in tokens])